forc/src/utils/defaults.rs

```diff
@@ -30,10 +30,12 @@ edition = "2021"
 license = "Apache-2.0"
 
 [dependencies]
-fuels = "0.15"
-fuels-abigen-macro = "0.15"
+fuels = {{ version = "0.16", features = ["fuel-core-lib"] }}
 tokio = {{ version = "1.12", features = ["rt", "macros"] }}
 
+[features]
+fuel-core-lib = ["fuels/fuel-core-lib"]
```

Comment:
Is the reason we don't have to specify `fuel-core-lib` as a Rust feature to `cargo test` a bug, or a feature? https://github.com/FuelLabs/fuels-rs/pull/419 is supposed to enable fuel-core bin as default instead of lib IIUC. If it's a bug, then these two lines should be removed.